app/Exceptions/BadUrlException.php

```diff
@@ -3,7 +3,4 @@
 
 class FreshRSS_BadUrl_Exception extends FreshRSS_Feed_Exception {
 
-	public function __construct(string $url) {
-		parent::__construct('`' . $url . '` is not a valid URL');
```

Comment:
Why removed?

Reply:
Never used

Reply:
never used, dead code

Reply:
It is used. See
https://github.com/FreshRSS/FreshRSS/blob/93c102d7eab69fee76f50fe207f9d3f4022372ca/app/Models/Feed.php#L274-L275

Please this PR focussed on the syntax update, with as little as possible of unrelated changes

Reply:
As a bit of an aside and depending on the specifics, available for extensions can also equal unused.